extensions/indexes/lucene/test/src/xquery/lucene/binary.xql

```diff
@@ -65,4 +65,12 @@ function luct:check-visibility-collection-pass() {
     system:as-user("admin", "", 
         ft:search("/db/lucenetest/sub/", "title:admin")/search/@uri/string()
     )
+};
+
+declare
+    %test:assertEquals("Lorem")
```

Comment:
A very useful addition! One question: wouldn't this test return the URI `/db/lucenetest/test.txt` rather than the contents of the field?

Reply:
true, interesting that the test passes...

Reply:
I bet the failure is being masked by https://github.com/eXist-db/exist/pull/2140.

Reply:
@joewiz maybe…

Reply:
@joewiz Likely indeed that #2140 has to be fixed first

Reply:
@tuurma Would you be able to get @wolfgangmm to take a look at #2140 first please?